tests/test_algebra_cascade.py

```diff
@@ -43,6 +43,11 @@ def generate_onnx_graph(dim, nbnode, input_name='X1', opv=None):
                np.array([[93., 93., 93., 93., 93.]]),
                np.array([[100100., 100100., 100100., 100100., 100100.]])]
         for opv in ({'': 10}, None, 9, 10, 11, onnx_opset_version()):
+            if isinstance(opv, dict):
+                if opv[''] > get_latest_tested_opset_version():
+                    continue
+            elif opv is not None and opv > get_latest_tested_opset_version():
```

Comment:
You may just write `opv` instead of `opv is not None`.

Reply:
It is not exactly the same. bool([] is not None) is True and bool([]) is False.

Reply:
Yes, but opv isn't a list here. It's either None or int value. Also, even if we had list here, wouldn't you want a bool([]) to be considered False?

Reply:
True, it is an integer but the comment is the same.